src/hotspot/cpu/x86/x86.ad

```diff
@@ -5859,9 +5859,9 @@ instruct vshiftB(vec dst, vec src, vec shift, vec tmp, rRegI scratch) %{
 
 instruct vshiftB_imm(vec dst, vec src, immI8 shift, vec tmp, rRegI scratch) %{
   predicate(vector_length(n) <= 8);
-  match(Set dst ( LShiftVB src shift));
-  match(Set dst ( RShiftVB src shift));
-  match(Set dst (URShiftVB src shift));
+  match(Set dst (LShiftVB src (LShiftCntV shift)));
+  match(Set dst (RShiftVB src (RShiftCntV shift)));
+  match(Set dst (URShiftVB src (RShiftCntV shift)));
```

Comment:
Regarding to this issue, it's no need to add these rules here. There is the vector shift rules already.

Reply:
The same to all other rules added in this file.

Reply:
> Please update the copyright year of `"vectorIntrinsics.cpp"` to 2021. Thanks!

Thank you. I will change that.

Reply:
> The same to all other rules added in this file.

OK. I will remove those codes.